package.json

```diff
@@ -39,7 +39,7 @@
     "electron-builder": "5.2.1",
     "electron-connect": "^0.3.7",
     "electron-packager": "^7.0.1",
-    "electron-prebuilt": "1.2.6",
+    "electron-prebuilt": "1.3.1",
```

Comment:
1.2.6 won't work, needs to be 1.2.7 at least.
Sorry, I think I mixed up this PR with the other one.